docs/rbd-nbd.md

```diff
@@ -29,6 +29,9 @@ client-side, which is inside the `csi-rbdplugin` node plugin.
 To use the rbd-nbd mounter for RBD-backed PVs, set `mounter` to `rbd-nbd`
 in the StorageClass.
 
+Please note that the minimum recommonded kernel version to use rbd-nbd is
+5.4 or higher.
```

Comment:
add a link to the ceph nbd doc where its mentioned for reference?

Reply:
I don't think they have mentioned it anywhere so far, I go to know this from the RBD-NBD maintainers Ilya in one of our PR conversations.

Reply:
Here is the link to conversation: https://github.com/ceph/ceph-csi/pull/2394#issuecomment-903636068

Reply:
@pkalever may be summarize that comment in a bit in the commit or as a note or for some reference in future ?

Reply:
Done now